packages/gatsby-plugin-google-gtag/README.md

```diff
@@ -0,0 +1,116 @@
+# gatsby-plugin-google-gtag
+
+Easily add Google Global Site Tag to your Gatsby site.
+
+## Install
+
+`npm install --save gatsby-plugin-google-gtag`
+
+## How to use
+
+```javascript
+// In your gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-plugin-google-gtag`,
+      options: {
+        // You can add multiple tracking ids and a pageview event will be fired for all of them.
+        trackingIds: [
+          'GA-TRACKING_ID', // Google Analytics / GA
+          'AW-CONVERSION_ID', // Google Ads / Adwords / AW
+          'DC-FLOODIGHT_ID', // Marketing Platform advertising products (Display & Video 360, Search Ads 360, and Campaign Manager)
+        ],
+        // This object gets passed directly to the gtag config command
+        // This config will be shared accross all trackingIds
+        gtagConfig: {
+          optimize_id: 'OPT_CONTAINER_ID',
+          anonymize_ip: true,
+        },
+        // This object is used for configuration specific to this plugin
+        pluginConfig: {
```

Comment:
@pieh do we have a standard for this? i.e. plugin specific options that are separate from the _service_ you're configuring?

Reply:
AFAIK there is no convention for that, it's totally up to plugins. 

In this instance I'm not sure if there is value in grouping those settings in `pluginConfig` object  - whole `options` is basically `pluginConfig`, but I don't mind having it this way as long as it's documented.